modules/ROOT/pages/fd-project-history.adoc

```diff
@@ -0,0 +1,31 @@
+= Discover What’s Changed in a Project
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+You can see a history of changes you make to a project in Flow Designer.
+The saved states are listed by the name of the person who made the change, time/date and activity, and the saved state name depends on the activity you were doing at the time the save occurred.
+
+Being able to view a saved state of a project is useful if you want to:
+
+* Detect what is or is not working between two different versions. For example, you started with a working, tested flow but you see errors and you want to revert back to the working state.
```

Comment:
"* Detect what is or is not working between two different versions. For example, you started with a working, tested flow but you see errors and you want to revert back to the working state."
I'm not sure what you mean by "between two different versions." Versions don't interact with each other, so how can things work or not work "between" them?
Why would I want to "detect what is...working"? If something is working, would I want to revert to another project version?
Do you mean "but now you see errors and you want to revert..."?

Reply:
Rewritten to address your comment.